.github/workflows/python-release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Set version
         run: python -m poetry version "${{ inputs.version }}"
         working-directory: ./python
-        if: "${{ github.event.inputs.version != 'master' }}"
+        if: "${{ github.event.inputs.version != 'main' }}"
```

Comment:
This is going to break CI? Is there a way one of the committers can commit this and change the main branch themselves to avoid affecting ongoing PRs?

Reply:
I rebased to remove the python resources (as they are now on `iceberg-python` repo).